v2/src/views/ViewTokenDetails.vue

```diff
@@ -70,7 +76,7 @@ watch(route, () => {
         state.holderAddress = query.holder as string
     } else {
         state.isHolder = false
-        state.holderAddress = undefined
+        state.holderAddress = ''
     }
     window.scrollTo(0, 0)
```

Comment:
right now the widnow scroll is envoked everytime the user is clicking on the 'holders' or 'transfers' tab. can you make sure it does not do  that